utils/src/test/kotlin/ExtensionsTest.kt

```diff
@@ -45,6 +45,40 @@ class ExtensionsTest : WordSpec({
         }
     }
 
+    "File.isSymbolicLink" should {
```

Comment:
Consider moving these tests into a separate file, e.g. `FileExtensionsTest`.
As result you'd have tests with `similar setup` in a `separate file` and thus the test `setup` and `tearDown` can be moved into the dedicated methods which leads to less code redundancy an better readability.

Reply:
Solved differently by moving setup and teardown code to the outer scope.